<launch>
  <arg name="exec_time" default="0"/>
  <include file="$(find pr2_description)/robots/upload_pr2.launch"/>
  <rosparam command="load" file="$(find pr2_cartesian_controllers)/config/controller_config.yaml"/>
  <node name="ects_sim" pkg="pr2_algorithms" type="test_bed_ects" args="$(arg exec_time)" output="screen">
    <rosparam param="left_arm_pose">[0.509, 0.326, -0.020, 0.198, 0.241, -0.655, 0.688]</rosparam>
    <rosparam param="right_arm_pose">[0.513, -0.177, 0.007, 0.670, 0.672, -0.234, 0.214]</rosparam>
  </node>
</launch>
